atom/browser/api/atom_api_auto_updater.h

```diff
@@ -32,6 +32,8 @@ class AutoUpdater : public mate::EventEmitter<AutoUpdater>,
 
   // Delegate implementations.
   void OnError(const std::string& error) override;
+  void OnError(const std::string& message,
+    const int code, const std::string& domain);
```

Comment:
normally we try to line up this line with the start of the params on the line above, thought that was actually a linting rule